### PR TITLE
Add setup moble version skeleton

### PR DESCRIPTION
**Pull request by BasitKorede**
# ADD SETUP AND PROJECT SKELETON.
  This adds the project setup and it uses a mobile-first approach.
### This pull request adds the following.
- It consists of the `Nav-bar` (navigation bar).
- The about me section.
- The language, skill, and framework section.
- And various social media platforms where  I can be contacted.

### Screenshot

<details>
    <summary>Click to view screenshot</summary>

screenshot

</details>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,13 +4,181 @@
...
+      </svg>
+      <svg
+        class="ellipse-19 position-absolute"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <g filter="url(#filter0_f)">
+          <circle cx="84" cy="84" r="34" fill="#EDECFF" />
+        </g>
+        <defs>
+          <filter
```
Comment (micheaol) [inline index.html:62]: Please ensure that you must stick to the design as much as possible (e.g. font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Your Implementation | Figma Design
:---:|:---:
your_design | figma_design

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @BasitKorede , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️

- _Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (BasitKorede) [reply index.html:63]: Hello sir I have made changes to this before creating a pull request please help me check well. Thanks

Comment (micheaol) [reply index.html:62]: @BasitKorede , you can call me Michael. Kindly re-submit for review on your dashboard.

Comment (BasitKorede) [reply index.html:62]: Thanks, @micheaol for the review and catching the bug. 👋 

It seems I broke the page by linking the wrong CSS file, [it is fixed now and should work as expected](https://raw.githack.com/BasitKorede/my-professional-portfolio/add-setup-moble-version-skeleton/index.html).

I will resubmit for another review. Thanks again! 🙂

**Review by gemmen29 — APPROVED** [review]
Hi @BasitKorede, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by BasitKorede** [comment]
Hello 👋 @micheaol thanks for the review 👍🏽

**Pull request merged**